examples/keycloak-config-cli-using-spire/README.md

```diff
@@ -0,0 +1,81 @@
+# keycloak-config-cli using spire
+
```

Comment:
Probably need to add a warning here for now, that this example requires Kubernetes 1.29+ or a Kubernetes version 1.28 with the SidecarContainers feature gate enabled.

Reply:
Not sure what you mean with this. Is this related to the `spire` chart, `keycloak` chart or some k8s feature?

Reply:
This example is using the sidecar k8s sidecar container feature (initContainer restart=always) which only exists in certain (very new) versions of k8s. If not mentioned, users of the example may get really confused when it doesn't work on their cluster thats older.

Reply:
I didn't know this. Thanks. Just tested it without it and noticed the `java-spiffe-helper` uses a [keepRunning](https://github.com/spiffe/java-spiffe/blob/main/java-spiffe-helper/src/main/java/io/spiffe/helper/cli/Runner.java#L42) flag. So yeah, will add a note in the readme.

Reply:
Done.